siac/rentercmd.go

```diff
@@ -294,6 +301,35 @@ func rentercontractscmd() {
 	w.Flush()
 }
 
+// rentercontractdetailscmd is the handler for the command `siac renter contracts <id>`.
+// It lists details of a specific contract.
+func rentercontractdetailscmd(id string) {
+	var rc modules.RenterContract
+	if err := getAPI("/renter/contract/"+id, &rc); err != nil {
+		die("Could not get contract details: ", err)
+	}
+
+	// Print overview.
+	var out = ""
+	out = fmt.Sprintf("%s\n\nOVERVIEW\n========", out)
+	out = fmt.Sprintf("%s\nEnd height:\t\t%v", out, rc.EndHeight())
+	out = fmt.Sprintf("%s\nHost:\t\t\t%v", out, rc.NetAddress)
+	out = fmt.Sprintf("%s\nID:\t\t\t%v", out, rc.ID)
+
+	// File Contract.
+	out = fmt.Sprintf("%s\n\nFILE CONTRACT\n=============", out)
+	out = fmt.Sprintf("%s\nFile Size:\t\t%v", out, filesizeUnits(int64(rc.FileContract.FileSize)))
+	// TODO: finish this section.
+
+	// Spending.
+	out = fmt.Sprintf("%s\n\nSPENDING\n========", out)
+	out = fmt.Sprintf("%s\nRenter funds:\t\t%s", out, currencyUnits(rc.RenterFunds()))
+	out = fmt.Sprintf("%s\nUpload Spending:\t%v", out, currencyUnits(rc.UploadSpending))
+	// TODO: finish this section.
+
+	fmt.Println(out)
```

Comment:
I think I would like to see:

```go
fmt.Printf(`
Contract %v
Host: %v (Public Key: %v)

Start Height: %v
End Height:   %v (%v blocks remaining)

Total Cost:        %v (Fees: %v)
Funds Allocated:   %v
Upload Spending:   %v
Storage Spending:  %v
Download Spending: %v
Remaining Funds:   %v

File Size: %v
`, args...)
```

Example:
```
Contract xxxxxxxxxx7c56f90b7cc0a8ef31c27792bd2bb8cc4d5c468ae5426bec5
Host: x.x.x.x:9982 (Public Key: ed25519:pubkeypubkeypubkey...)

Start Height: 103456
End Height:   123456 (20000 blocks remaining)

Total Cost:        2 KS (Fees: 500 SC)
Funds Allocated:   1.5 KS
Upload Spending:   300 SC
Storage Spending:  200 SC
Download Spending: 200 SC
Remaining Funds:   700 SC

File Size: 10 GB
```
@vevarm, please chime in with what sort of output you'd like to see.